MCprep_addon/materials/skin.py

```diff
@@ -335,7 +344,29 @@ def download_user(self, context: Context, username: str) -> Optional[Path]:
 		self.track_param = "username"
 	return saveloc
 
+def check_entity_texture(texture_path: str) -> Optional[Path]:
+	context = bpy.context
+	addon_prefs = util.get_user_preferences(context)
+	active_pack = bpy.path.abspath(context.scene.mcprep_texturepack_path)
+	active_pack = os.path.join(
+		active_pack, "assets", "minecraft", "textures", "entity")
+
+	base_pack = bpy.path.abspath(addon_prefs.custom_texturepack_path)
+	base_pack = os.path.join(
+		base_pack, "assets", "minecraft", "textures", "entity")
+
+	# if not os.path.isdir(active_pack):
+	# 	env.log(f"No models found for active path {active_pack}")
+	# 	return None
+	base_has_textures = os.path.isdir(base_pack)
```

Comment:
I'm not really the biggest fan of `os.path` for filepath operations, it's all string manipulation (vs `pathlib.Path` which is much nicer in abstraction). Not a blocker, but I'd advise using `pathlib` instead of `os.path` for new code since it's much better, and works with `os.path` functions anyway if needed.